src/addPanGestureHandling.tsx

```diff
@@ -39,6 +39,7 @@ export const addPanGestureHandling: AddPanGestureHandling = ({
     addAnimatedGestureHandler(
       {
         onStart: ({translationX, translationY}) => (_, context) => {
+          'worklet'
```

Comment:
Since we aren't running our own compilation through the Babel plugin (at least currently), these need to be manually tagged as worklets